_layouts/home.html

```diff
@@ -12,7 +12,7 @@
                 </div>
                 <div class="hero-spotlight-header hero-spotlight-header-home">
                     <h1>The effortless open-source tool for event
-                        organisation, archival and collaboration
+                        organization, archival and collaboration
```

Comment:
Pretty sure this is intentional since the site uses en_GB (see #18)

Reply:
Indeed. CERN rule: use "Organization" for CERN, "organisation" for everything else.

Reply:
Got it - thanks :)